app/components/nextflow/samplesheet/sample_cell_component.html.erb

```diff
@@ -1,4 +1,4 @@
 <td class="px-6 py-3 text-left whitespace-nowrap">
   <%= sample.name %>
-  <%= fields.hidden_field :sample, value: "Sample_#{sample.id}" %>
+  <%= fields.hidden_field :sample, value: "Sample_#{sample.puid}" %>
```

Comment:
**nitpick (blocking):** The value should just be the puid, no prefix

Reply:
Sorry, updated in 905a875